app/uk/gov/hmrc/eusubsidycompliancestub/services/EisService.scala

```diff
@@ -26,10 +26,10 @@ import java.time.LocalDate
 object EisService {
 
   private val SectorLimits = Map[Sector, IndustrySectorLimit](
```

Comment:
sectorLimits should not be defined in 2 different places. Can we please remove one. We can use the same one everywhere

Reply:
I Agree 👍🏻